lib/Runtime/Library/JSON.cpp

```diff
@@ -706,25 +706,26 @@ namespace JSON
 
         if(isEmpty)
         {
-            result = scriptContext->GetLibrary()->CreateStringFromCppLiteral(_u("{}"));
+            result = scriptContext->GetPropertyString(PropertyIds::object_Empty);
```

Comment:
Just note that a PropertyString is useful only if it is used for property look up with o[x].